package.json

```diff
@@ -9,6 +9,8 @@
     "@types/enzyme": "3.10.3",
     "@types/history": "^4.7.3",
     "@types/jest": "24.0.17",
+    "@types/lodash": "^4.14.144",
```

Comment:
We normally use fixed versions, please remove the `^` at the beginning.

Reply:
Thanks! Noted for future contributions. Fixed on commit 1d2ecb1.